DQMOffline/CalibTracker/plugins/SiStripQualityHotStripIdentifierRoot.cc

```diff
@@ -201,7 +201,7 @@ SiStripBadStrip* SiStripQualityHotStripIdentifierRoot::getNewObject(){
 void SiStripQualityHotStripIdentifierRoot::algoBeginRun(const edm::Run& iRun,const edm::EventSetup& iSetup){
   //Retrieve tracker topology from geometry
   edm::ESHandle<TrackerTopology> tTopoHandle;
-  iSetup.get<TrackerTopologyRcd>().get(tTopoHandle);
+  iSetup.get<IdealGeometryRecord>().get(tTopoHandle);
   tTopo = tTopoHandle.product();
```

Comment:
@suchandradutta why this changes? I dont' see them in the corresponding commit in the 7.6.X branch b33a00a845324e8b544f79fee4f3db89d91aa423. I suspect this causes the crash the test observe.